os/tools/mkbootparam.py

```diff
@@ -30,11 +30,13 @@
 # This script generates Boot Parameters for kernel.
 #
 # Boot Parameter information :
-# +---------------------------------------------------------------------+
-# | Active Idx |  First Address  | Second Address  |      Reserved      |
-# |  (1 byte)  |    (4 bytes)    |    (4 bytes)    | (8Kbytes - 9bytes) |
-# +---------------------------------------------------------------------+
+# +--------------------------------------------------------------------------------------------+
+# |   CRC32  |   Size   | Active Idx |  First Address  | Second Address  |       Reserved      |
```

Comment:
`CRC32` is a method of checksum, so I think the description can be `checksum` instead of `CRC32`.